docs/modules/ROOT/pages/how-tos/prepare_for_modulesync.adoc

```diff
@@ -0,0 +1,33 @@
+= Make your existing component ready for ModuleSync
+
+* Create `.sync.yml` with the following content:
++
+[source,yaml]
+----
+:global:
+  componentName: my-component <1>
+
+docs/antora.yml:
+  name: my-component <1>
+  title: My Component <2>
+
+LICENSE:
+  year: 2021 <3>
+----
+<1> The component name
+<2> A more human friendly component name
+<3> The year the component got created (see existing `LICENSE` file)
+
+* Ensure the rules from the previous section are followed
+* Check the https://github.com/projectsyn/commodore/tree/master/commodore/component-template[component template] and the https://github.com/projectsyn/modulesync-control[ModuleSync template] for further things of interest.
+
+
+[NOTE]
+====
+You can locally test the impact of ModuleSync on an existing component.
+
+* Clone https://github.com/projectsyn/modulesync-control[modulesync-control]
+* Add your component to `managed_modules.yml` (no need to commit/push)
+* Run `make sync:noop`
```

Comment:
Maybe mention that this runs in Docker.

Reply:
Right. At least it's documented in makefile somewhat:
```
help                 Show this help
sync:noop            Syncs the managed modules locally but doesn't create PRs. Requires running SSH agent with access to github.com, ~/.ssh/known_hosts, ~/.gitconfig.
sync:offline         Same as sync:noop, but doesn't clone/pull from git repos. Use this target to test the modulesync template.
```